### PR TITLE
Enable smoke test for ARM arch

### PR DESCRIPTION
**Pull request by JorTurFer**
Signed-off-by: Jorge Turrado <jorge_turrado@hotmail.es>

**Note:** This PR uses my fork of kind-action because they don't support ARM64 arch yet. Till [this PR](https://github.com/helm/kind-action/pull/55) is not merged, using mine (or another fork at kedacore side) is needed

**Note 2:** This PR also cleans up some unused imports on e2e tests

### Checklist

- [x] Commits are signed with Developer Certificate of Origin (DCO - [learn more](https://github.com/kedacore/keda/blob/main/CONTRIBUTING.md#developer-certificate-of-origin-signing-your-work))
- [x] Tests have been added
- [x] A PR is opened to update our Helm chart ([repo](https://github.com/kedacore/charts)) *(if applicable, ie. when deployment manifests are modified)*
- [x] A PR is opened to update the documentation on ([repo](https://github.com/kedacore/keda-docs)) *(if applicable)*
- [x] Changelog has been updated and is aligned with our [changelog requirements](https://github.com/kedacore/keda/blob/main/CONTRIBUTING.md#Changelog)

Related https://github.com/kedacore/keda/issues/2262

### CODE REVIEW

File: tests/run-arm-smoke-tests.sh [hunk 1]
```diff
@@ -27,18 +28,52 @@ function run_tests {
     do
         test_case="scalers/${scaler}"
         counter=$((counter+1))
-        ./node_modules/.bin/ava $test_case > "${test_case}.log" 2>&1 &
+        ./node_modules/.bin/ava $test_case > "${test_case}.1.log" 2>&1 &
```
Comment (zroubalik) [inline tests/run-arm-smoke-tests.sh:31]: Do we need to do this? Isn't enough just to rename the retry test logs?

**Review by zroubalik — COMMENTED** [review]
Looking good! Great job

File: tests/run-arm-smoke-tests.sh (hunk 1 above)
Comment (JorTurFer) [reply tests/run-arm-smoke-tests.sh:31]: It's the same approach as normal e2e test, but I can change in both sides.
Do you prefer something like 'retry-xxxxxxx.test.ts'?

Comment (zroubalik) [reply tests/run-arm-smoke-tests.sh:31]: Yeah 👍

**Pull request merged**